extern/sector-storage/ffiwrapper/sealer_test.go

```diff
@@ -544,6 +544,7 @@ func TestSealAndVerifyAggregate(t *testing.T) {
 
 	avi.Proof, err = ProofProver.AggregateSealProofs(avi, toAggregate)
 	require.NoError(t, err)
+	require.Len(t, avi.Proof, 11188)
```

Comment:
This number is less than 11220 which we expect in the mock impl, but I'm going to assume that that's caused by the fact that 2k sectors have 10x less snarks